multiversx_sdk/abi/explicit_enum_value.py

```diff
@@ -0,0 +1,11 @@
+from typing import Any
+
+from multiversx_sdk.abi.string_value import StringValue
+
+
+class ExplicitEnumValue(StringValue):
+    def __init__(self, value: str = "") -> None:
+        self.value = value
+
+    def __eq__(self, other: Any) -> bool:
+        return isinstance(other, ExplicitEnumValue) and self.value == other.value
```

Comment:
Currently, we don't differentiate between different types of enums if they have the same value. We'll brainstorm this a bit (could be perfectly fine in practice).